packages/@react-spectrum/dialog/docs/Dialog.mdx

```diff
@@ -43,52 +47,355 @@ import {Heading, Text} from '@react-spectrum/typography';
       <Header><Heading>The Heading</Heading></Header>
       <Divider />
       <Content>
-        <Text>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sit amet tristique risus. In sit amet suscipit lorem. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In condimentum imperdiet metus non condimentum. Duis eu velit et quam accumsan tempus at id velit. Duis elementum elementum purus, id tempus mauris posuere a. Nunc vestibulum sapien pellentesque lectus commodo ornare.</Text>
+        <Text>
+          This is a dialog.
+        </Text>
       </Content>
-      <Footer>
+      <ButtonGroup>
         <Button variant="secondary" onPress={close}>Cancel</Button>
         <Button variant="cta" onPress={close}>Confirm</Button>
-      </Footer>
+      </ButtonGroup>
     </Dialog>
   )}
 </DialogTrigger>
 ```
 
 ## Content
 
-*If the component has a children prop that accepts any type of content (e.g. `ReactNode`), include this section. Please include a note about how to internationalize the content.*
+A standard Dialog has the following anatomy:
+
+<svg xmlns="http://www.w3.org/2000/svg" width="611.999" height="372" viewBox="0 0 611.999 372">
```

Comment:
Right now I'm putting the entire svg into the mdx file since markdown image embedding (aka `![alt text](FILE PATH "ALT TEXT")`)doesn't seem to render the image quite right.

I can move this huge block into a local React component instead and render that instead and I'm playing around with making a generic ExampleImage component to put into the docs package that can accepts a svg input as a prop, just having difficulties getting Parcel to recognize the import

Reply:
Actually might be able to use `<Image>` instead...